regression/mint/react-icu/package.json

```diff
@@ -2,6 +2,6 @@
     "name": "i18next-demo",
     "dependencies": {
         "@l10nmonster/helpers": "^1",
-        "@l10nmonster/helpers-json": "^1"
+        "@l10nmonster/helpers-json": "../../../helpers-json"
```

Comment:
@diegone could this be the solution for the problem we talked about, that the regression tests require npm modules to be published first?

Reply:
that will make it work locally but it won't test whether it'll break clients that use npm

Reply:
So I guess we want to execute tests using the code in the `dist` or somewhere else that has artifacts of a build?